GitUI/Editor/FileViewerInternal.cs

```diff
@@ -36,6 +36,14 @@ public partial class FileViewerInternal : GitModuleControl, IFileViewer
 
         public FileViewerInternal()
         {
+            Disposed += (sender, e) =>
+            {
+                //// _diffHighlightService not disposable
+                //// _lineNumbersControl not disposable
+                //// _currentViewPositionCache not disposable
+                _findAndReplaceForm.Dispose();
+            };
```

Comment:
💡 This should be registered after the `Initialize` methods.

Reply:
Why? In order to be safe, the cleanup should be registered before something can go wrong.

Reply:
If something goes wrong in the constructor (i.e. an exception is thrown), this event will not be called.

Reply:
If you like. Moved after the `Initialize` methods and rebased.